### DISCUSSION TITLE
Ideavim disables IDE built-in searching inside Project (file) tab

### DISCUSSION THREAD

**Discussion by Zackhardtoname** [JetBrains/ideavim#658] [Q&A] [answered]
Without ideavim, you could press keystrokes to move to folders/files with that prefix, but with ideavim, you get weird results.

Any idea to mitigate that? Maybe disabling ideavim just for the Project tab somehow?

**Comment by citizenmatt** [comment, marked as answer]
IdeaVim shouldn't be affecting the Project tab, unless you have the NERDTree extension enabled, in which case, you need to use NERDTree shortcuts such as `/` to search. Try disabling the extension in your `~/.ideavimrc`
